server/controllers/auth.js

```diff
@@ -109,7 +110,10 @@ exports.loadUser = asyncHandler(async (req, res, next) => {
         name: user.name,
         email: user.email
       },
-      profile
+      profile: {
```

Comment:
Why don't we return the entire profile instead

Reply:
I returned the entire profile.